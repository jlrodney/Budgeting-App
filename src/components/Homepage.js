import React from 'react';


const Homepage = () => (
  <div>
    This will be the homepage!
  </div>
);

export default Homepage;
