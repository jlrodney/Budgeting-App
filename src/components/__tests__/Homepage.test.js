import React from 'react';
import { shallow } from 'enzyme';
import Homepage from '../Homepage';

describe('Homepage should render  correctly', () => {
  it('should have a div', () => {
    const wrapper = shallow(<Homepage />);
    expect(wrapper.find('div').text()).toEqual("This will be the homepage!");
  })
});
describe('should render Header correctly', () => {
  const wrapper = shallow(<Homepage />);
  expect(wrapper).toMatchSnapshot();
})
