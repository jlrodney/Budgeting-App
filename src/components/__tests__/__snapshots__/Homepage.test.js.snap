// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  This will be the homepage!
</div>
`;
